x/precisebank/keeper/invariants.go

```diff
@@ -121,3 +127,28 @@ func BalancedFractionalTotalInvariant(k Keeper) sdk.Invariant {
 		), broken
 	}
 }
+
+// FractionalDenomNotInBankInvariant checks that the bank does not hold any
+// fractional balances. These assets, e.g. akava, should only exist in the
+// x/precisebank module and shouldn't be "real" assets managed by x/bank.
```

Comment:
can we add a note that the reason it's not allowed isn't because the assets aren't "real", but because the akava are backed by ukava already tracked by x/bank and including them would result in double counting (i guess except in the case where there is a non-zero remainder?)